tested/main.py

```diff
@@ -30,10 +33,19 @@ def run(config: DodonaConfig, judge_output: IO):
     _, ext = os.path.splitext(config.test_suite)
     is_yaml = ext.lower() in (".yaml", ".yml")
     if is_yaml:
+        if language:
+            textual_suite, missing_keys = run_translation_with_str(
```

Comment:
I would make this method return `preprocessor_messages` instead of `missing_keys`